client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-confirmation.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button } from '@automattic/components';
+import { NavigatorHeader } from '@automattic/onboarding';
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalUseNavigator as useNavigator,
+} from '@wordpress/components';
+import { Icon, image, verse, layout } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import NavigatorTitle from './navigator-title';
+import './screen-confirmation.scss';
+
+interface Props {
+	onConfirm: () => void;
+}
+
+const ScreenConfirmation = ( { onConfirm }: Props ) => {
+	const translate = useTranslate();
+	const { goBack } = useNavigator();
+
+	return (
+		<>
+			<NavigatorHeader
+				title={ <NavigatorTitle title={ translate( 'Great job!' ) } /> }
+				description={ translate( 'Prepare to bring your site to life with your unique content.' ) }
+				onBack={ () => {
+					goBack();
+					// @TODO: Track back event
+				} }
+			/>
+			<div className="screen-container__body">
+				<VStack spacing="4" className="screen-confirmation__list">
+					<HStack spacing="4" alignment="top">
+						<div className="screen-confirmation__list-item-icon">
```

Comment:
[nit] How about making the list into an array and then mapping them into the JSX elements?

```js
const list = [
  {
    icon: image,
    title: translate( 'Upload your photos' ),
    description: translate( 'Morbi pellentesque mauris eget laoreet.' )
  },
  ...
];

{list.map((...) => (
  ...
)}
```

Reply:
Thanks